style(Table): remove hover on zebra colored row

Barsnes opened the pull request:
table with zebra enabled made zebra colored rows white on hover. Removed this.
This does not need a changeset, since the changes to table has not been pushed to next yet

mimarz commented:
> table with zebra enabled made zebra colored rows white on hover. Removed this. This does not need a changeset, since the changes to table has not been pushed to next yet

Its removed in comparison to `main` so shouldn't it be a changeset then?

Barsnes commented:
> > table with zebra enabled made zebra colored rows white on hover. Removed this. This does not need a changeset, since the changes to table has not been pushed to next yet
> 
> Its removed in comparison to `main` so shouldn't it be a changeset then?

no, this was an expected change in a past PR about removing hover on table. This had a changeset in that PR

mimarz commented:
> > > table with zebra enabled made zebra colored rows white on hover. Removed this. This does not need a changeset, since the changes to table has not been pushed to next yet
> > 
> > 
> > Its removed in comparison to `main` so shouldn't it be a changeset then?
> 
> no, this was an expected change in a past PR (#2285) about removing hover on table. This had a changeset in that PR

ah, this was missed when we removed hover?

Barsnes commented:
> > > > table with zebra enabled made zebra colored rows white on hover. Removed this. This does not need a changeset, since the changes to table has not been pushed to next yet
> > > 
> > > 
> > > Its removed in comparison to `main` so shouldn't it be a changeset then?
> > 
> > 
> > no, this was an expected change in a past PR (#2285) about removing hover on table. This had a changeset in that PR
> 
> ah, this was missed when we removed hover?

yes, this is a fix for this, Would have made a changeset if we had released a next version in between this and that PR :)
